R/calculateDMM.R

```diff
@@ -68,14 +68,12 @@
 #' se <- SummarizedExperiment(assays = list(counts = counts),
 #'                            colData = colData)
 #'
-#'
-#' #
 #' dmn <- calculateDMN(se)
 #' dmn[[1L]]
 #'
 #' # since this take a bit of resources to calculate for k > 1, the data is
 #' # loaded
-#' \dontrun{
+#' \donttest{
```

Comment:
Beaware, all these change will making testing task very long.

Reply:
This was changed because this causes a NOTE in BiocCheck and change was recommended. Hence I would keep to change but we can also change back if necessary.